javascript/javascript2/week2/02-woking-with-movies/script.js

```diff
@@ -0,0 +1,106 @@
+//Q1) short title--------------------------------
+const shortTitledMovies = movies.filter((movie) => movie.title.length < 5);
+console.log(shortTitledMovies.length);
+
+// Q2) long movie titles---------------------------
+const longMovieTitles = movies.filter((movie) => movie.title.length > 5);
+// console.log(`movies with long titles ${JSON.stringify(longMovieTitles)}`);
+
+// Q3)made between 1980-1989-------------------------------
+
+let number = 0;
+const moviesMade = movies.filter(
+  (movie) => movie.year >= 1980 && movie.year <= 1989
+);
+// console.log(`made between 1980-1989 ${JSON.stringify(moviesMade)}`);
+
+//Q4) extra key called tag---------------------------------------
+const tag = movies.map(function (movie) {
```

Comment:
Here you're using an anonymous function instead of an arrow function, is that on purpose, do you understand how they are different? 

I'd recommend sticking to one syntax for consistency :)